hack/golangci.yaml.in

```diff
@@ -15,163 +15,169 @@ run:
   timeout: 30m
 
 output:
-  sort-results: true
+  show-stats: false
+
+version: "2"
+
+formatters:
+  exclusions:
+    paths:
+      - third_party
 
 issues:
   max-issues-per-linter: 0
   max-same-issues: 0
 
-  # The default excludes disable the "should have comment or be unexported" check from revive.
-  # We want that to be enabled, therefore we have to disable all default excludes and
-  # add those back one-by-one that we want. See https://github.com/golangci/golangci-lint/issues/456#issuecomment-617470264
-  exclude-use-default: false
-  exclude:
-    # staticcheck: Developers tend to write in C-style with an explicit 'break' in a 'switch', so it's ok to ignore
-    - ineffective break statement. Did you mean to break out of the outer loop
-
-  exclude-files:
-    - "^zz_generated.*"
-
-  # Excluding configuration per-path, per-linter, per-text and per-source
-  exclude-rules:
-    # exclude ineffassign linter for generated files for conversion
-    - path: conversion\.go
-      linters:
-        - ineffassign
-
-    # SSA Extract calls are allowed in tests.
-    - linters:
-        - forbidigo
-      text: should not be used because managedFields was removed
-      path: _test.go$
-
-    # Adding unversioned feature gates is allowed in tests
-    - linters:
-        - forbidigo
-      text: should not use Add, use AddVersioned instead
-      path: _test.go$
-
-    {{- if .Base}}
-
-    # TODO(oscr) Remove these excluded directories and fix findings. Due to large amount of findings in different components
-    # with different owners it's hard to fix everything in a single pr. This will therefore be done in multiple prs.
-    - path: (pkg/volume/*|test/*|azure/*|pkg/cmd/wait*|request/bearertoken/*|metrics/*|filters/*)
-      linters:
-        - gocritic
-    {{- end}}
+linters:
+  exclusions:
```

Comment:
@mmorel-35: did you perhaps use `linters.exclusions.warn-unused` to check for unused exclusion entries?

I am wondering whether we should enable that by default. We to keep the exceptions as minimal as possible.